packages/clay-card/src/Description.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+import {CardDescriptionDisplayType} from './types';
+
+interface CardDescriptionProps extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactText;
+
+	/**
+	 * Truncates the text inside a description.
+	 */
+	truncate?: boolean;
+
+	/**
+	 * Type of description that can be applied for a text.
+	 */
+	type: CardDescriptionDisplayType;
+}
+
+export type CardDescriptionType = React.FunctionComponent<CardDescriptionProps>;
+
+export const Description: CardDescriptionType = ({
+	children,
+	className,
+	truncate = true,
+	type,
+	...otherProps
+}) => {
+	const {displayType} = React.useContext(Context);
```

Comment:
Using both `displayType` and `type` will probably get confusing. Can we use something else for `type`?

Reply:
When refactoring displayType horizontal and interactive, I can use displayType properly on ClayCard.Description.
See https://github.com/liferay/clay/pull/2007/commits/150e7a7a69d47049f3cd583f60549534fb91719a